Automattic/wp-calypso
# Disable pagination when page is loaded with pattern hash

Fixes https://github.com/Automattic/dotcom-forge/issues/6391

## Proposed Changes

When the pattern library is opened with a single-pattern hash, this PR makes it so that pagination is disabled altogether to prevent scenarios where the targeted pattern is hidden.

## Testing Instructions



1. Navigate to `/patterns/about`
2. Click on `Load {num} more patterns`
3. Copy the permalink for one of the patterns shown after clicking load more
4. Open the link and ensure that you are automatically scrolled to the correct pattern
